pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -321,14 +328,27 @@ export class ConfigurationCard extends React.Component {
     }
 
     render() {
+        // We use a Privileged component for its ability to
+        // conditionally show a tooltip, even when the button is not
+        // actually disabled, so the "allowed" property really means
+        // "does not have tooltip" here.
+
+        const hostname_tooltip = (this.permission.allowed == false
```

Comment:
Please use `===` for `false` comparison.

Reply:
Done, thanks.